### PR TITLE
user authentication

### PR DESCRIPTION
**Pull request by VuTranAnh**
user log in, log out

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: dùng .present?

Comment (VuTranAnh) [inline app/helpers/sessions_helper.rb]: da c

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.present?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: !current_user.nil?  == current_user.present?

Comment (VuTranAnh) [inline app/helpers/sessions_helper.rb]: à e nhầm c, ẩu quá

**Pull request merged**